site/source/cookbook/modern-web-apps/efficient-state-management.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: docs
+category: modern-web-apps
+title: Manage state in your application
+overview: Manage state to decrease complexity in your application
+---
+
+## Objective
+
+This recipe demonstrates a technique for managing state in your Dojo 2 application.
+
+## Procedure
+
+1. Create a widget which uses a property from `this.properties`:
+
+```js
+render() {
+    return v('div', [this.properties.message]);
+}
+```
+
+2. You created a widget in step one, wrap this widget with a `Container` by creating the file `src/containers/YourWidgetContainer.ts` and adding the following code:
```

Comment:
You created a widget in step one, wrap this widget with -> Wrap the widget created in step one with

Reply:
Done!